### PR TITLE
➖ remove general settings

### LINKED ISSUE #5983: Remove all setting from general tab
Screenshot 2023-05-15 at 13 08 13

Move button `CLEAR CACHE & RELOAD` to Metadata tab

### PR DESCRIPTION
**Pull request by roiLeo**
I like to delete stuff

## PR Type

- [x] Refactoring

## Context

- [x] Closes #5983 
- [x] Vue3 component

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

## Screenshot 📸

Capture d’écran 2023-05-15 à 4 15 50 PM

### <samp>🤖 Generated by Copilot at 4ca5004</samp>

Simplified the settings page by removing unused components and tabs, and added a cache-clearing button to the metadata component for easier debugging.

### <samp>🤖 Generated by Copilot at 4ca5004</samp>

> _The settings page had a makeover_
> _No more tabs for general or `metadata`_
> _But don't worry, you can still discover_
> _A button to clear the cache and reset_
> _It's handy for testing and debugging, you bet_

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Review by daiagi — APPROVED** [review]
lgtm

File: components/metadata/Metadata.vue [hunk 1]
```diff
@@ -17,9 +17,16 @@
...
+    <NeoButton @click="refresh"> Clear Cache & Reload </NeoButton>
   </div>
 </template>
 
 <script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+
 const { chainProperties } = useChain()
+const refresh = () => {
+  window.sessionStorage.removeItem('vuex')
```
Comment (preschian) [inline components/metadata/Metadata.vue:29]: what if using `clearSession()` in `utils/cachingStrategy`?

**Review by preschian — APPROVED** [review]
otherwise lgtm

File: components/metadata/Metadata.vue (hunk 1 above)
Comment (roiLeo) [reply components/metadata/Metadata.vue:29]: because it would clear/remove user session (like a logout)

**Comment by vikiival** [comment]
> I like to delete stuff

should this go on tshirt?

**Pull request merged**